src/main/java/core/basesyntax/strategy/DecreaseAmountOperationHandlerImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitRecord;
+
+public class DecreaseAmountOperationHandlerImpl implements OperationHandler {
+    @Override
+    public int getChangedAmount(FruitRecord record) {
+        int amount = Storage.getStorage().get(record.getFruitName());
```

Comment:
Why do we need to create here new variable "amount"?

Reply:
fixed